lib/datasource/go/index.ts

```diff
@@ -1,5 +1,5 @@
 import URL from 'url';
-import { PLATFORM_TYPE_GITLAB } from '../../constants/platforms';
+import { PlatformID } from '../../constants/platforms';
```

Comment:
I think all constants should be imported via `lib/constants` re-exports, so we can use a single import. We do this for most `lib/types` already.

@JamieMagee WDYT?

Reply:
Yeah, I agree.